cgp/node.py

```diff
@@ -33,7 +33,7 @@ class Node:
 
     _arity: int
     _active: bool = False
-    _inputs: List[int]
+    _input_nodes: List[int]
```

Comment:
hmm, shouldn't this be renamed in many other places as well? like in `Node.__init__`?

Reply:
Yes, thanks, I changed this in the other places.